### DISCUSSION TITLE
ubuntu 20.04 on chrome os

### DISCUSSION THREAD

**Discussion by george-code** [dnschneid/crouton#4906] [Q&A]
hey, I was wondering if anyone knew how to install crouton, but with Ubuntu 20.04 i still haven't found an answer.

-George-code

**Comment by DennisLfromGA** [comment]
Ubuntu 20.04 is Focal Fossa. If you run `crouton -r list` you'll see that focal has an asterisk after it which means:  
> Releases marked with * are unsupported, but may work with some effort.

So you could try it using:  
- `sudo crouton -r focal -t ...`    

Then add the targets you want to install.

*Hope this helps,*
-**DennisLfromGA**

**Reply by Diamondbot-ramfs** [reply]
> Ubuntu 20.04 is Focal Fossa. If you run `crouton -r list` you'll see that focal has an asterisk after it which means:
> 
> > Releases marked with * are unsupported, but may work with some effort.
> 
> So you could try it using:
> 
>     * `sudo crouton -r focal -t ...`
> 
> 
> Then add the targets you want to install.
> 
> _Hope this helps,_ -**DennisLfromGA**

Focal is not supported by Crouton, try the install **RED letters not supported.** it may work it may fail
